readthedocs/rtd_tests/utils.py

```diff
@@ -69,6 +69,14 @@ def make_test_git():
     log.info(check_output(['git', 'add', '.'], env=env))
     log.info(check_output(['git', 'commit', '-m"Add submodule"'], env=env))
 
+    # Add a relative submodule URL in the relativesubmodule branch
+    log.info(check_output(['git', 'checkout', '-b', 'relativesubmodule', 'master'], env=env))
+    log.info(check_output(
+        ['git', 'submodule', 'add', '-b', 'master', './', 'relativesubmodule'],
+        env=env
+    ))
+    log.info(check_output(['git', 'add', '.'], env=env))
+    log.info(check_output(['git', 'commit', '-m"Add relative submodule"'], env=env))
     # Add an invalid submodule URL in the invalidsubmodule branch
     log.info(check_output(['git', 'checkout', '-b', 'invalidsubmodule', 'master'], env=env))
```

Comment:
Now this `invalidsubmodule` is no more invalid.

We probably need to remove this.